Update MSI-enabled resource types

johndowns opened the pull request:
Add API Management to the list of Azure resource types that support managed service identities.

azurecla commented:
Hi __@TaSato-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@TaSato-MSFT : Thanks for your contribution to the Azure documentation! The author, @ganesr, has been notified to review your proposed change.

ganesr commented:
This change does not add any value. Please close the PR.

TaSato-MSFT commented:
OK.

johndowns committed:
Update MSI-enabled resource types

Add API Management to the list of Azure resource types that support managed service identities.

johndowns committed:
Fix typo
